Pipfile

```diff
@@ -9,12 +9,10 @@ crcmod = "==1.7"
 future = "==0.17.1"
 protobuf = "==3.15.3"
 psutil = "==5.7.0"
-numpy = "==1.18.4"
-tensorflow = "==2.3.0"
```

Comment:
How will testcase generation work if this is removed - https://github.com/google/clusterfuzz/blob/481c78b28fb2d9a401a6f3fe302746a8f6110954/src/python/bot/fuzzers/ml/rnn/generate.py#L21 ?

Reply:
Good point! I mistakenly thought we were installing this somewhere else in the Docker process, but that's using this Pipfile as well. 

I added a line to install tensorflow in the base image separately from this Pipfile install.

Reply:
might be still a problem for chromium mac,android bots, but we can even disable it there ? thoughts like try catch it.

Reply:
I added a try/catch to the place importing the generator to log_warn and exit if we can't import it.